centos_ci.sh

```diff
@@ -292,14 +292,40 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  DOC_USER=$(echo $REPO | cut -d"/" -f4)
+  cd gopath/src/github.com/minishift/minishift
+
+  # Fetch remote repo and checkout to specified branch if not default minishift repo
+  if [[ "$DOC_USER" != "minishift" ]]; then
+    git remote add remote-user $REPO
+    git fetch remote-user
+    git checkout -b remote-docs-branch --track remote-user/$BRANCH
+  fi
+
+  git log -n 1 # Check last commit
+
+  install_docs_prerequisite_packages;
+  make gen_adoc_tar
+  build_openshift_origin_docs $(pwd)/docs/build/minishift-adoc.tar;
+
+  mkdir -p minishift/docs/ondemand/$DOC_USER-$BRANCH
+  cp -r openshift-docs/_preview/openshift-origin minishift/docs/ondemand/$DOC_USER-$BRANCH/ # Copy the openshift-origin
+  cp docs/build/minishift-adoc.tar minishift/docs/ondemand/$DOC_USER-$BRANCH/ # Copy doc tar
+  # http://stackoverflow.com/a/22908437/1120530; Using --relative as --rsync-path not working
+  RSYNC_PASSWORD=$RSYNC_PASSWORD rsync -av --delete --relative minishift/docs/ondemand/$DOC_USER-$BRANCH/ minishift@artifacts.ci.centos.org::minishift/
```

Comment:
So where does this publish to? This is not even a single destination, right? This publishes to repo name and branch? Where is this documented?